### PR TITLE
ASUI-44: Deployment resources and dependency examples for console

### PR DESCRIPTION
**Pull request by MikeEdgar**
These are (currently) straight copies of the upstream resources, without the Bash installation scripts.

### CODE REVIEW

**Comment by MikeEdgar** [comment]
@PaulRMellor, is there a need to include an additional README or other form of documentation alongside these new resources?

**Comment by PaulRMellor** [comment]
> @PaulRMellor, is there a need to include an additional README or other form of documentation alongside these new resources?

I think it would help Mike. A line on what each file is for, say. 
Streams includes this for the examples in the download zip: https://github.com/strimzi/strimzi-kafka-operator/tree/main/examples

File: examples/console/resources/console/console.deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        - containerPort: 8080
+        env:
+        - name: KAFKA_SECURITY_PROTOCOL
+          value: SASL_SSL
+        - name: KAFKA_SASL_MECHANISM
+          value: SCRAM-SHA-512
+        - name: CONSOLE_KAFKA_EXAMPLE
+          value: ${NAMESPACE}/console-kafka
+        - name: CONSOLE_KAFKA_EXAMPLE_BOOTSTRAP_SERVERS
+          value: bootstrap.console-kafka.apps-crc.testing:443
```
Comment (k-wall) [inline examples/console/resources/console/console.deployment.yaml:34]: Code Ready Containers?

**Review by k-wall — APPROVED** [review]
looks reasonable to me.

File: examples/console/resources/console/console.deployment.yaml (hunk 1 above)
Comment (MikeEdgar) [reply examples/console/resources/console/console.deployment.yaml]: Good catch, thank you. I've updated this to use a parameter, matching the bootstrap host in `resources/kafka/console-kafka.kafka.yaml`.

**Comment by kyguy** [comment]
@MikeEdgar let me know when this is ready and we can merge it

**Review by jankalinic — APPROVED** [review]

**Comment by MikeEdgar** [comment]
@kyguy - conflicts resolved. Please merge any time.

**Pull request merged**